src/core/lib/gprpp/memory.h

```diff
@@ -27,18 +27,27 @@
 
 namespace grpc_core {
 
+// The alignment of memory returned by gpr_malloc().
+#define ALIGNMENT_FOR_DEFAULT_ALLOCATION 8
```

Comment:
we could use a `constexpr` for this. We should also specify the units as part of the name.

Reply:
Done.